core/src/main/java/org/bitcoinj/params/UnitTestParams.java

```diff
@@ -36,7 +36,7 @@ public UnitTestParams() {
         id = ID_UNITTESTNET;
 
         targetTimespan = 200000000;  // 6 years. Just a very big number.
-        maxTarget = new BigInteger("00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 16);
+        maxTarget = new BigInteger("007fffff0000000000000000000000000000000000000000000000000000000000", 16);
```

Comment:
Perhaps add a comment explaining this magic number represents EASIEST_DIFFICULTY_TARGET? A simple end-of-line comment like the one above would do I'd say.

Reply:
@schildbach I added the comment and force-pushed.

Of course, this change is a precursor to actually using a `maxTarget` generated from `EASIEST_DIFFICULTY_TARGET`.